apps/ledger-live-mobile/scripts/mobile-env-md.mjs

```diff
@@ -14,7 +14,7 @@ function main() {
 }
 
 function gen() {
-  let md = `<!-- this file is generated by mobile-env-md.test.ts -->\n`;
+  let md = `<!-- this file is generated by mobile-env-md.mjs -->\n`;
```

Comment:
unrelated fix - this reference was wrong, which meant extra time tracking down why this was causing build failures